### ISSUE TITLE
Problems compiling Canu 1.5

### ISSUE THREAD

**Issue by bistace** [marbl/canu#477]
Hi,
I downloaded the Canu 1.5 source code but when I try to compile it, I get an error. Here is the command that produces the error : 

```
cc -o /env/export/source/src/Canu/canu/Linux-amd64/obj/libcanu.a/AS_UTL/libbacktrace/dwarf.o -c -MD -I/env/export/source/src/Canu/canu/src -IAS_UTL -Istores -Istores/libsnappy -Ialignment -Iutgcns/libNDalign -Iutgcns/libcns -Iutgcns/libpbutgcns -Iutgcns/libNDFalcon -Iutgcns/libboost -Imeryl/libleaff -IoverlapInCore -IoverlapInCore/libedlib -IoverlapInCore/liboverlap -Ifalcon_sense/libfalcon AS_UTL/libbacktrace/dwarf.c
```

and here is the error itself : 
```
AS_UTL/libbacktrace/dwarf.c: In function ‘dwarf_lookup_pc’:
AS_UTL/libbacktrace/dwarf.c:2779: erreur: ‘__ATOMIC_ACQUIRE’ undeclared (first use in this function)
AS_UTL/libbacktrace/dwarf.c:2779: erreur: (Each undeclared identifier is reported only once
AS_UTL/libbacktrace/dwarf.c:2779: erreur: for each function it appears in.)
AS_UTL/libbacktrace/dwarf.c:2839: erreur: ‘__ATOMIC_RELEASE’ undeclared (first use in this function)
AS_UTL/libbacktrace/dwarf.c: In function ‘dwarf_fileline’:
AS_UTL/libbacktrace/dwarf.c:2974: erreur: ‘__ATOMIC_ACQUIRE’ undeclared (first use in this function)
AS_UTL/libbacktrace/dwarf.c: In function ‘backtrace_dwarf_add’:
AS_UTL/libbacktrace/dwarf.c:3107: erreur: ‘__ATOMIC_ACQUIRE’ undeclared (first use in this function)
```

Do you have any idea on how I could sort out this problem ?

Thanks a lot

**Comment by brianwalenz** [comment]
Your compiler is too old.  If possible upgrade to gcc 4.8.5 or 4.9.4.  This error occurs with the ancient 4.4 series compiler (the last release was five years ago) in some Linux distributions.

You can build with this compiler, `make BUILDSTACKTRACE=0`, but this will disable a very useful crash log, so don't crash, or if you do crash, don't report it.

Duplicate of #425, #438, and #446, but your report is more succinct.

Leaving this open in the hope that the Makefile will detect and disable automagically.

**Comment by bistace** [comment]
Thank you for your fast answer! Indeed, I changed my compiler for gcc-4.9 and it worked out fine.

**Comment by ekg** [comment]
I ran into this problem. The build detected `cc` as my C compiler, which is an old version, while my actual gcc was new (v5). I solved it by running `CC=gcc CXX=g++ make`.
